Source/Scene/Expression.js

```diff
@@ -1028,6 +1028,18 @@ define([
         var type = this._type;
         var value = this._value;
 
+        // Right may be a string if it's a member variable: e.g. "${property.name}"
+        var stringMember = typeof(this._right) === 'string';
+        if (stringMember) {
+            //>>includeStart('debug', pragmas.debug);
+            throw new DeveloperError('Error generating style shader: string members are not supported.');
+            //>>includeEnd('debug');
+        }
+        if (stringMember) {
+            // Avoid jsHint error by creating a new block. Want to return undefined in release.
+            return undefined;
+        }
```

Comment:
In release I want to return undefined, but placing it right after the throw creates a jsHint error. Any tips to avoid this?